packages/kotti-input-number/src/InputNumber.vue

```diff
@@ -124,7 +127,14 @@ export default {
 		},
 	},
 	created() {
-		this.currentValue = typeof this.value === 'number' && !Number.isNaN(this.value) ? this.value : 0
+		this.currentValue =
+			typeof this.value === 'number' && !Number.isNaN(this.value)
+				? this.value
+				: 0
+		if (this.disabled) {
```

Comment:
IIRC you had a watcher on this.disabled, so using `immediate` flag on the watcher can make sure this happens so u don't add the same code to the created @Goldziher